code/modules/antagonists/clockcult/clock_helpers/slab_abilities.dm

```diff
@@ -40,7 +40,7 @@
 		else if(L.stat)
 			to_chat(ranged_ability_user, "<span class='neovgre'>\"There is use in shackling the dead, but for examples.\"</span>")
 			return TRUE
-		else if(L.handcuffed)
+		else if (L.handcuffed && istype(L.handcuffed,/obj/item/restraints/handcuffs/clockwork))
```

Comment:
why don't you just have the istype check since it would fail if you're not handcuffed?

Reply:
I assumed it would break if L.handcuffed was null, so it's an oversight.